test/androidTest/java/org/thoughtcrime/securesms/GetFakeVerifiedTest.java

```diff
@@ -36,17 +34,27 @@
 @RunWith(AndroidJUnit4.class)
 public class GetFakeVerifiedTest {
 
+    @Rule
+    public GrantPermissionRule permissionRule = GrantPermissionRule.grant(Manifest.permission.READ_CONTACTS,
+                                                                          Manifest.permission.READ_PHONE_STATE,
```

Comment:
Will this help us avoid having to manually accept popups the way we currently do now?

Reply:
Yes, it automatically accepts all the native permission dialogs.